fvm/crypto/hash.go

```diff
@@ -17,38 +20,36 @@ import (
 type prefixedHashing struct {
 	hash.Hasher
 	usePrefix bool
-	tag       [flow.DomainTagLength]byte
+	tag       [tagLength]byte
 }
 
 // paddedDomainTag converts a string into a padded byte array.
-func paddedDomainTag(s string) ([flow.DomainTagLength]byte, error) {
-	var tag [flow.DomainTagLength]byte
-	if len(s) > flow.DomainTagLength {
-		return tag, fmt.Errorf("domain tag %s cannot be longer than %d characters", s, flow.DomainTagLength)
+func paddedDomainTag(s string) ([tagLength]byte, error) {
+	var tag [tagLength]byte
+	if len(s) > tagLength {
+		return tag, fmt.Errorf("domain tag cannot be longer than %d characters, got %d", tagLength, len(s))
```

Comment:
The error no longer includes the tag text. Is this ok?

Reply:
I will add it back, that's better.